src/truststore/_macos.py

```diff
@@ -431,7 +431,41 @@ def _verify_peercerts_impl(
         # We always want system certificates.
         Security.SecTrustSetAnchorCertificatesOnly(trust, False)
 
+        # if _mac_version_info <= (10, 13):
+        if 1 or _mac_version_info <= (10, 13):
+            sec_trust_result = Security.SecTrustResultType()
+            Security.SecTrustEvaluate(trust, ctypes.byref(sec_trust_result))
+
+            try:
+                sec_trust_result = int(sec_trust_result)
+            except (TypeError, ValueError):
+                sec_trust_result = -1
```

Comment:
I tried the code on macOS 10.12 where I was encountering the pip issues and it almost worked. Thanks for working on this!

440 needs to get changed to use `sec_trust_result.value`, otherwise it will be -1 (due to except block).

For `c_uint` types, `.value is documented to be `int`, so maybe that `int()` is not even needed if the result type is always `c_uint`. Please check, I am not familiar with that stuff.

Reply:
After that change, `pip install ...` worked!

Reply:
It looks like that might be the reason for the github CI failing.

Reply:
Ping @illume - can you try the fix I suggested?